draft-ietf-tls-tls13.md

```diff
@@ -931,8 +931,8 @@ Subsequent Handshake:
 {: #tls-resumption-psk title="Message flow for resumption and PSK"}
 
 As the server is authenticating via a PSK, it does not send a
-Certificate or a CertificateVerify. When a client offers resumption
-via PSK it SHOULD also supply a "key_share" extension to the server as well
+Certificate or a CertificateVerify message. When a client offers resumption
+via PSK, it SHOULD also supply a "key_share" extension to the server as well
```

Comment:
I think this sentence is fine with or without this comma.  (I might add one before "to allow", though.)  But I have no real opinion and leave it to the editor's discretion.